net-misc/youtube-viewer/youtube-viewer-3.1.9.ebuild

```diff
@@ -7,7 +7,7 @@ EAPI=5
 inherit gnome2-utils eutils perl-module vcs-snapshot
 
 DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
-HOMEPAGE="https://trizen.googlecode.com"
+HOMEPAGE="http://trizenx.blogspot.nl/2012/03/gtk-youtube-viewer.html"
```

Comment:
seems so, based in the Netherlands here

I guess a more canonical extension should be used (.com?).